### ISSUE TITLE
copyright?

### ISSUE THREAD

**Issue by nthieberger** [Language-Research-Technology/language-data-commons-vocabs#15]
Do we need this or is it already used elsewhere and does not need to be in a language data commons metadata set? 
Copyright status (copyrighted, public domain, unknown)

**Comment by ptsefton** [comment]
Schema.org has these copyright related props 

![image](https://github.com/Language-Research-Technology/language-data-commons-vocabs/assets/756184/ab8f96a5-2e56-4e30-abda-fa06462226fe)

We could have a set of defined terms or just put some text in copyrightNotice -- eg copyrightNotice: "The copyright status of this work is unknown" (or whatever)

**Comment by ptsefton** [comment]
Or put it in copyrightHolder

**Comment by nthieberger** [comment]
Yes, I looked at those but neither is about copyright status, one is about
the notice, the other is about the holder

**Comment by nthieberger** [comment]
PREMIS. copyrightStatus  https://www.archivematica.org/en/docs/archivematica-1.9/user-manual/metadata/premis/#copyright-status

**Comment by ptsefton** [comment]
That looks suitable, can be added to the vocab


----------------------------

Dr Peter Sefton
Senior Technical Advisor, School of Languages and Culture
Mobile: 0404 096 932

________________________________
